Commands/utils.py

```
@@ -0,0 +1,212 @@
+    if nick not in state:
+        allowed = True
+    else:
+        allowed = (time.time() - state[nick]) > cooldown
+
+    if allowed:
+        state[nick] = time.time()
+        return True
+    
+    return False
+
+def clean_str(text: str, remove: list[str] = []) -> str:
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)